src/styles/blocks/product.scss

```diff
@@ -0,0 +1,120 @@
+.product-card {
+  box-sizing: border-box;
+
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+
+  width: $product-card-width;
+
+  border: 1px solid $product-border-color;
+  border-radius: $product-border-radius;
+
+  &__image {
+    box-sizing: border-box;
+    background-image: url('../images/imac.jpeg');
+    background-repeat: no-repeat;
+    background-size: 100% 100%;
+    width: 160px;
+    height: 134px;
+    margin: 32px 19px 0;
+  }
+
+  &__content {
+    margin: 40px 16px 16px;
+
+    font-family: $product-font-family, sans-serif;
```

Comment:
Ensure that the SCSS variable `$product-font-family` is defined in your project. Undefined variables will cause a compilation error.